test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -177,6 +177,7 @@ class test_ucp_sockaddr : public ucp_test {
     }
 
     void get_sockaddr() {
+        std::set<std::pair<sa_family_t, std::string>> added;
         std::vector<ucs::sock_addr_storage> saddrs;
```

Comment:
since we later use either the first added or a randomized index, i would keep like that?

Reply:
it could be like:
```
auto i = saddrs.begin();
std::advance(i, rand_index);
```
but I'm fine with both approaches since it's just local calculation in test